pkgs/top-level/all-packages.nix

```diff
@@ -6514,18 +6514,7 @@ with pkgs;
 
     m17n = callPackage ../tools/inputmethods/ibus-engines/ibus-m17n { };
 
-    inherit mozc;
-
-    mozc-ut = mozc.override { dictionaries = [
-      mozcdic-ut-alt-cannadic
-      mozcdic-ut-edict2
-      mozcdic-ut-jawiki
-      mozcdic-ut-neologd
-      mozcdic-ut-personal-names
-      mozcdic-ut-place-names
-      mozcdic-ut-skk-jisyo
-      mozcdic-ut-sudachidict
-    ]; };
+    inherit mozc mozc-ut;
```

Comment:
Maybe this can be dropped as well? I believe they should be picked up by the `pkgs/by-name` system automatically

Reply:
I think users will generally expect all ibus engines to live inside `pkgs.ibus-engines`, so it's probably fine to have it duplicated there. For example, the docs uses the namespace to generate a list of available engines:

https://github.com/NixOS/nixpkgs/blob/71a33cc7af3a199871b8f04e5f1e991e9ca11267/nixos/modules/i18n/input-method/ibus.nix#L41-L46

Reply:
Ah right, I was a bit blinded by the diff. It looked like you were inheriting it straight into top-level. This is good :+1: